.github/ci/files/var/classes/DataObject/Car.php

```diff
@@ -0,0 +1,1020 @@
+<?php
```

Comment:
do we really need that? 
can't we just exclude `DummyDataCommand` from the checks? 
that command should be removed anyway in the longrun...